src/python/ddapp/tabledemo.py

```diff
@@ -82,7 +83,7 @@ def __init__(self, robotStateModel, playbackRobotModel, ikPlanner, manipPlanner,
 
         self.constraintSet = []
 
-        self.reachDist = 0.07
+        self.reachDist = 0.12
```

Comment:
No, not anymore since we switched to using Marco's collision free reaching. I will change it nonetheless and add a commit in a bit